src/components/layout/RegisterNameForm/RegisterNameForm.tsx

```diff
@@ -0,0 +1,159 @@
+                onChange={(e) => handleTargetID(e.target.value)}
+                maxLength={43}
+                style={
+                  targetID && isArweaveTransactionID(targetID)
+                    ? { border: '4px solid var(--success-green)' }
+                    : targetID && isArweaveTransactionID(targetID)
+                    ? { border: '4px solid var(--error-red)' }
+                    : {}
+                }
+              />
+            ) : antFlow ===
+              'I have an Arweave Name Token (ANT) I want to use' ? (
+              <Dropdown
+                showChevron={true}
+                showSelected={false}
+                options={['ant one', 'ant two', 'ant three', 'ant four']}
+                selected={antAddress}
+                setSelected={setAntAddress}
+                headerElement={
+                  <SearchBar
+                    validationPredicate={(name) =>
+                      isArNSDomainNameValid({ name })
+                    }
+                    successPredicate={() => {
+                      return true;
+                    }}
+                    placeholderText="Enter ANT Contract ID"
+                    height={30}
+                  />
+                }
+                footerElement={
+                  <div
+                    className="data-input last-dropdown-option center"
+                    style={{ background: 'var(--bg-color)' }}
+                  >
+                    <span className="text bold white center">
+                      See All Tokens
+                    </span>
+                  </div>
+                }
+              />
+            ) : (
+              <></>
+            )}
+          </div>
+          {/**add a tag component for multiple controllers seperated by "," commas. In the future might have an address book. Possibly suggest other accounts from arconnect. */}
+          <input
+            className="data-input center"
+            type="text"
+            placeholder="Controllers"
+            onChange={(e) => handleTargetID(e.target.value)}
+            maxLength={43}
+          />
+          <div className="input-group center">
+            <input
+              className="data-input center"
+              type="text"
+              placeholder="Nickname*"
+            />
+            <input
+              className="data-input center"
+              type="text"
+              placeholder="Ticker*"
+            />
+            <Dropdown
+              showSelected={true}
+              showChevron={true}
+              selected={ttl}
+              setSelected={(value) =>
+                updateRegisterState({ key: 'setTTL', value })
+              }
+              options={{
+                '100s': 100,
+                '200s': 200,
+                '300s': 300,
+                '400s': 400,
+                '500s': 500,
+                '600s': 600,
+                '700s': 700,
+                '800s': 800,
```

Comment:
change `s` to `secs` - i know this probably conflicts with design but it's not clear what this is setting

Reply:
changed, but this *should* be getting changed to a counter comp in a future pr